types/wlr_wl_shell.c

```diff
@@ -108,10 +109,10 @@ static void wl_shell_destroy(struct wl_resource *resource) {
 	}
```

Comment:
It can't belong to multiple lists, but it can be taken out and put somewhere else - which is what I'm worrying about.
Let's just say users shouldn't do that and I'll unlink directly